src/components/Calendar.js

```diff
@@ -47,24 +56,62 @@ class Calendar extends React.Component {
   };
 
   _handlePreviousClick = () => {
-    const currentDate = moment.unix(this.state.currentDate).startOf('month').subtract(1, 'm').unix();
+    const currentDate = moment
+      .unix(this.state.currentDate)
+      .startOf('month')
+      .subtract(1, 'm')
+      .unix();
 
     this.setState({
       currentDate
     });
   };
 
+
+  _handleDayKeyDown = (e) => {
+    const startDate = moment.unix(this.state.currentDate).startOf('month').startOf('week');
+    const endDate = moment.unix(this.state.currentDate).endOf('month').endOf('week');
+
+    if (keycode(e) === 'enter') {
+      this._handleDateSelect(this.state.focusedDay, e);
+    }
+
+    if (['left', 'right', 'up', 'down'].includes(keycode(e))) {
+      //prevent browser scrolling with arrow key press
+      e.preventDefault();
+
+      const day = __calculateDayByKey(keycode(e), this.state.focusedDay);
```

Comment:
yeah, seems like passing in the moment obj may be a win since you've already got to take into consideration moment here. Better to isolate the places that need moment.

Reply:
so have it take a third argument, `moment`?